src/platform/Linux/ConnectivityUtils.h

```diff
@@ -28,6 +28,169 @@
 #include <linux/types.h> /* for "caddr_t" et al      */
 #include <linux/wireless.h>
 
+/* Status codes (IEEE Std 802.11-2016, 9.4.1.9, Table 9-46) */
+#define WLAN_STATUS_SUCCESS 0
```

Comment:
Could we place these into some `WlanConstants.h` file or similar? The 'Utils'  naming is very generic and it is not immediately obvious that `ConnectivityUtils.h` will contain status codes and reasons for use with GLIB callbacks.

Reply:
Acked